README.md

```diff
@@ -56,8 +60,54 @@ Dependencies
 ------------
 None
 
+Configuring a check
+-------------------
+
+To configue a check you need to add an entry to the `datadog_checks` section.
+The first level key is the name of the check and the value is the yaml payload
+to write the the configuration file.
+
+Example:
+
+**Process check**
+
+We define 2 instances for the `process` check.
+This will create:
+- for agent6: `/etc/datadog-agent/conf.d/process/conf.yaml`
+- for agent5: `/etc/datadog-agent/conf.d/process.yaml`
+
+```
```

Comment:
nit: if you add `yml` after the triple backticks, github will use yaml syntax coloring on the code block :)